client/src/components/MainContainer.jsx

```diff
@@ -0,0 +1,18 @@
+import React, { useState } from "react";
+import CardsContainer from "./CardsContainer";
+import VideoForm from "./VideoForm";
+import fullData from "../exampleresponse.json";
+
+function MainContainer() {
+    const [videoData, setVideoData] = useState(fullData);
+    
+    console.log("Video Data from MainContainer---> ", videoData);
```

Comment:
When you push to the origin (on the cloud), it's worth looking over your codebase to check if there are any redundant development-level code lines scattered somewhere. For example, in most cases, console.log() helps us debug our program during the development phase (when we are actually coding and building the program/app). It's best if you remove any console.logs from your program to declutter the codebase.

Reply:
I decluttered them. Thank you for your feedback!